website/src/views/buckets.js

```diff
@@ -16,10 +16,10 @@ export default function Buckets(){
         api.get('/bins').then((res) => {
             if(mounted){
                 let tempBins = [{ grade: res.data[0][1], range: `0-${res.data[0][0]}`}];
-                for(let i = 1; i < res.data.length; i++){
+                for (let i = 1; i < res.data.length; i++) {
                     const grade = res.data[i][1];
                     const range = `${+res.data[i - 1][0] + 1}-${res.data[i][0]}`;
-                    tempBins = [...tempBins, { grade, range }];
+                    tempBins = [{ grade, range }, ...tempBins];
```

Comment:
Not sure why it was written this way in the past, but this method of appending / prepending ends up being `O(n^2)` so it would be more efficient to `unshift` it.  The most efficient way to do this would probably be to iterate the other way and use the `push` method to construct the pre-reversed array, though between these two ops, the complexities are meaningfully negligible so do what you feel more comfortable with.